google_compute_engine/accounts/oslogin_utils.py

```diff
@@ -123,7 +123,7 @@ def UpdateOsLogin(self, enable, duration=NSS_CACHE_DURATION_SEC):
     self.update_time = current_time
     if enable:
       self.logger.info('Activating OS Login.')
-      return self._RunOsLoginControl('activate') or self._RunOsLoginNssCache()
+      return self._RunOsLoginControl('activate --twofactor' if two_factor else 'activate') or self._RunOsLoginNssCache()
     else:
       self.logger.info('Deactivating OS Login.')
       return (self._RunOsLoginControl('deactivate')
```

Comment:
We should update this function to take a string and a boolean for 2fa. It would be a lot more readable and easier to maintain in the future if we add other options.